### ISSUE TITLE
Footer icons lack text/accessible name

### ISSUE THREAD

**Issue by bkardell** [opencollective/opencollective#3366]
**Describe the bug**
The icons in the footer (blog, gh, twitter, etc) lack text alternatives and are not accessible.  There are numerous solutions to this, the easiest is to just include the actual text of the link, but other solutions could include adding an aria-label/title or a visually-hidden/sr-only element containing the text.

**To Reproduce**
1. Go to a collective page like https://opencollective.com/css-focus-visible-in-webkit-safari
2. Open dev tools and select one of these elements, choose the accessibility panel and see that there is no text/computed aria-label for this link (or use lighthouse and look at the report)

**Expected behavior**
It has text or a accessible computed label, at least.

**Screenshots**
![Screenshot from 2020-07-27 11-52-57](https://user-images.githubusercontent.com/870501/88563312-d8417500-cfff-11ea-8b59-891841bb948b.png)

Affects any device/OS/browser
